src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackHeaderBlock.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Slack\Block;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+
+/**
+ * @author Tomas Norkūnas <norkunas.tom@gmail.com>
+ */
```

Comment:
As this is a new class, shouldn't we mark it with:
```suggestion
 *
 * @experimental in 5.3
 */
```
❓